modules/node-app-service/variables.tf

```diff
@@ -37,6 +37,17 @@ variable "app_settings" {
   default     = {}
 }
 
+#EasyAuth setting
+variable "auth_config" {
+  description = "Config for the frontdoor in tooling subscription"

```

Comment:
Not sure this description matches what its for!

Reply:
Updated the description